src/Tools/ManifestGenerator/README.md

```diff
@@ -0,0 +1,4 @@
+# dotnet-roslyn-manifest-generator
+
+https://github.com/dotnet/roslyn/blob/efd69176d6ad7f9dbe1d87525201760a6c44e7a0/docs/compilers/terrapin.md#artifacts-manifest-file
```

Comment:
nit: is there a permanent link we could use instead?

Reply:
@jcouv Isn't this already a permanent link?

Reply:
Once Jared's changes in #49886 are merged, we will probably link to the document in-repo, possibly with a summary of the relevant section in here.